addons/contexts/src/shared/types.d.ts

```diff
@@ -21,11 +22,11 @@ export declare interface AddonSetting {
   icon?: ComponentProps<typeof Icons>['icon'] | '';
   title: string;
   components?: unknown[];
-  params?: Array<{
+  params?: {
     name: string;
     props: GenericProp;
     default?: boolean;
-  }>;
+  }[];
```

Comment:
Not sure about this one, I think `Array<>` is easier to read. What other devs are thinking about it?

Reply:
I agree personally on that I find `Array<X>` to be easier to read then `X[]`, but this is a recommended rule from the typescript folk